content/en/docs/release_notes/johannesburg.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Johannesburg release notes
+linkTitle: Johannesburg
+description: 25 March 2022
+date: 2022-03-25
+Hide_readingtime: true
+---
+## Summary
+
+In this release we officially shipped the OpenAPI flow-trigger and added the ability to disable response validation.
+
+{{% releasenotes/upgrade %}}
+
+## Breaking changes
+
+* #7320: [@axway/api-builder-plugin-ft-oas](https://www.npmjs.com/package/@axway/api-builder-plugin-ft-oas) now supports loading multiple flow-triggers.
+* #7217: New projects from the API Builder CLI are now initialised without example APIs, models, schema flows or endpoints.
+
+## Features
+
+* #7315: [@axway/api-builder-plugin-ft-oas](https://www.npmjs.com/package/@axway/api-builder-plugin-ft-oas) now supports a new option "Response validation" that allows toggling between "error", "warn" and "disabled" modes. When set to "warn" or "disabled" and the response does not match the specification, the app will not respond with 500 but will execute the flow and respond with the result of its execution. When set to "warn", validation will still occur, and be printed to the console.
+
+## Fixes
+
+* #7322: In [@axway/api-builder-plugin-ft-oas](https://www.npmjs.com/package/@axway/api-builder-plugin-ft-oas) added a log `warn` and `error` when fail to import OpenAPI specs that do not validate.
```

Comment:
Now logs the reason(s) for failing to validate OpenAPI specifications on import.